akka/akka-http-scala-seed.g8
# Provide Scala HTTP Seed project for Scala API

Issue: https://github.com/akka/akka-http/issues/1055
Use Variables for:
- Project name
- Akka HTTP version

Provides two Main classes:
- One with the common code shown in the docs
- One using `HttpApp` to provide a less verbose starting point
- Tests

Not (yet) part of this PR:
- Example usage of the client. I'm open to suggestions if this should be included or not

--------------------------
In order to test this, there are 2 options:
- With SBT on the template itself:
  - On the top directory, run `sbt test`.
  - This will create a new project under `target/g8/my-akka-http-project`.
  - Run `sbt test` on this newly created project
- The real run:
  - On any directory, run `sbt new jlprat/akka-http-scala-seed.g8` (my fork has this branch as default one)
  - Check that it prompts asking for:
    - Project name
    - Akka HTTP version (default is 10.0.6)
  - Step into this new project
  - Run `sbt test`
